src/lp/fixed_variable.jl

```diff
@@ -1,20 +1,105 @@
+@doc raw"""
+    FixedVariableRule
+
+Eliminate all fixed variables.
+
+## Presolve
+
+A variable ``x_{j}`` with lower and upper bounds ``l_{j}, u_{j}``
+is fixed to its lower bound if
+```math
+| l_{j} - u_{j}| \leq ϵ,
+```
+where ``\epsilon`` is a prescribed tolerance.
+
+The variable is then eliminated from the problem as follows: ``\forall i,`` row ``i``
+```math
+l^{r}_{i} \leq a_{i, j}x_{j} + \sum_{k \neq j} a_{i, k} x_{k} \leq u^{r}_{i}
+```
+is transformed into
+```math
+l^{r}_{i} - a_{i, j} l_{j} \leq \sum_{k \neq j} a_{i, k} x_{k} \leq u^{r}_{i} - a_{i, j} l_{j}.
+```
+
+## Integer variables
+
+This rule applies to integer and continuous variables.
```

Comment:
This sentence is confusing, given the subsection title. I would rename the subsection to something like "Inferring infeasibility".

Reply:
Also, I'm a bit confused by this statement. You can only infer infeasibility for integer variables, correct?

Reply:
I think I see why the confusion. The "rule" in
```
This rule applies to integer and continuous variables
```
referred to "FixedVariableRule".

What I meant (in more words) is: "the _Fixed variable reduction_ is applied to both integer and continuous variables, but when fixing an integer variable, the following infeasibility check is performed: [...]"

I'll update that part.